registration/src/main/proto/lobby/registration/conference.proto

```diff
@@ -0,0 +1,116 @@
+// A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+// OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+// SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+// LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+// DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+// THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+// (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+syntax = "proto3";
+
+package lobby.registration;
+
+option java_package="com.lobby.registration";
+option java_multiple_files = true;
+
+import "google/protobuf/timestamp.proto";
+
+// The conference created by a business customer.
+message Conference {
+
+    // The unique conference id.
+    ConferenceId conference_id = 1;
+
+    // The conference locator access code.
+    // Used by a business customer (conference owner) to view conference details.
+    // Generated when the business customer creates the conference.
+    string access_code = 2;
+
+    // The conference name. Used in the slug.
+    string name = 3;
+
+    // The conference description.
+    string description = 4;
+
+    // The part of the URL used to access the conference.
+    // Consists of conferenceId + conferenceName (name is cut).
+    string slug = 5;
+
+    // The start date of the conference.
+    google.protobuf.Timestamp start_date = 6;
+
+    // The end date of the conference (inclusive).
+    google.protobuf.Timestamp end_date = 7;
+
+    // The types of seats available for the conference.
+    repeated SeatType seat_type = 8;
+}
+
+// The id of the conference based on the UUID (universally unique identifier).
+message ConferenceId {
+    string uuid = 1;
+}
+
+// The type of a seat which represents the right to be admitted to a conference or
+// to access a specific session at the conference (tutorial, workshop etc).
+message SeatType {
+
+    // The unique seat type id.
+    SeatTypeId seat_type_id = 1;
+
+    // The conference id the seat associated with.
+    ConferenceId conference_id = 2;
+
+    // The seat name.
+    string name = 3;
+
+    // The seat description.
+    string description = 4;
+
+    // The seat price (in USD).
+    double price = 5;
+
+    // The total quantity of seats for the conference.
+    int32 quantity_total = 6;
+
+    // The available quantity of seats for the conference.
+    int32 quantity_available = 7;
+}
+
+// The id of the seat type based on the UUID (universally unique identifier).
+message SeatTypeId {
+    string uuid = 1;
+}
+
+// Associates an attendee with a seat in a confirmed order.
+message SeatAssignment {
+
+    // The seat type for the attendee.
+    SeatTypeId seat_type_id = 1;
+
+    // The personal info of the attendee.
+    PersonalInfo attendee = 2;
+}
+
+message PersonalInfo {
```

Comment:
We have a type for users see user.proto in java-core. Some of the fields may not be used, but I would re-use our standard type (and tune it, if needed) rather than re-invent it for the purpose of this sample application.